### ISSUE TITLE
Add-on vanished

### ISSUE THREAD

**Issue by KillerCroc1** [sebastiencs/rarbg-addon#13]
Hey man the content of your add-on has vanished and has not returned yet what's going on ?? Please fix as getting alot of fans asking me what's going on

**Comment by sebastiencs** [comment]
@KillerCroc1 Are you able to use the addon now ?

**Comment by KillerCroc1** [comment]
Nope just checked again no content is showing up at all from rarbg

**Comment by KillerCroc1** [comment]
Where did you go man I hope you are ok we all thought your add-on was dead

**Comment by sebastiencs** [comment]
I'm fine, thanks. I will investigate the issue

**Comment by KillerCroc1** [comment]
Awesome man happy to see you back as we have no good add-ons as we all depended on your epic add-on ;-)

**Comment by sebastiencs** [comment]
@KillerCroc1 can you test now ? It should work

**Comment by KillerCroc1** [comment]
It's is back woohoo what was the issue man it's so good to have you back

**Comment by KillerCroc1** [comment]
It was back now it's all gone again no rarbg sources

**Comment by KillerCroc1** [comment]
Your content is back up ;-) but it's not showing on all devices that now have stremio the android app now supports android TV ;-) your add-on is a no show on there any chance you could fix this ?

**Comment by KillerCroc1** [comment]
Hey man your add-on is not showing at all again please fix

**Comment by KillerCroc1** [comment]
Hey man need assistance I can get rarbg on my mobile but not on android tvim getting reports from all my fans that your add-on is not showing at all for them even the guys at stremio can't get rarbg content showing but I can 🤔

**Comment by KillerCroc1** [comment]
yeah i can confirm im getting your add-on on one device my android mobile windows is a no show for me and fans and android tv is the same please fix
